### PR TITLE
LPS-154570 do not show some dropdowns if is video shortcut

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
- LPS-154570 only show the option for download / checkin / checkout / cancel checkout actions if the file version has han allowed content type

image

image

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
I tried to make something similar to 
```

	@Override
	public Menu getMenu() throws PortalException {
		Menu menu = super.getMenu();

		DLVideoExternalShortcutUIItemsUtil.processUIItems(menu.getMenuItems());

		return menu;
	}
```

But I found no way to access to the data of the DropdownItem without changing the API

**Comment by AliciaGarciaGarcia** [comment]
@ambrinchaudhary this is the issue I found reviewing the https://github.com/liferay-lima/liferay-portal/pull/3073

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java [hunk 1]
```diff
@@ -1261,7 +1263,8 @@ public MenuItem getCheckinMenuItem() throws PortalException {
 	public boolean isCancelCheckoutActionAvailable() throws PortalException {
 		if ((_fileShortcut != null) ||
 			!_fileEntryDisplayContextHelper.
-				isCancelCheckoutDocumentActionAvailable()) {
+				isCancelCheckoutDocumentActionAvailable() ||
+			!_isAllowedFileVersionContentType()) {
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java:1267]: Adjectives go at the end in conditional sentences: _isFileVersionContentTypeAllowed().

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java:1267]: thanks!

**Comment by AliciaGarciaGarcia** [comment]
changes made and rebase

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**